crates/codegen/parser/runtime/src/support/parser_result.rs

```diff
@@ -177,3 +184,14 @@ impl NoMatch {
         Self { expected_tokens }
     }
 }
+
+#[derive(PartialEq, Eq, Clone, Debug)]
+pub struct SkippedUntil {
+    pub nodes: Vec<cst::Node>,
+    /// Skipped text following the last node
+    pub skipped: String,
+    /// At which token was the stream pointing at when we bailed
+    pub found: TokenKind,
+    /// Token we expected to find
+    pub expected_tokens: Vec<TokenKind>,
```

Comment:
This is a Vec to deduplicate code in the `parser_function.rs` but I can rework it so that it's only a single `TokenKind`

Reply:
Agreed; changed in 662693b6b6f4787cd577a2a02c2c34243776d4c2